DSP_Plugins.GalacticScale/DSP_Plugins.GalacticScale.csproj

```diff
@@ -38,7 +38,7 @@
       <HintPath>..\Libs\0Harmony.dll</HintPath>
     </Reference>
     <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>G:\SteamLibrary\steamapps\common\Dyson Sphere Program\DSPGAME_Data\Managed\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\..\Program Files (x86)\Steam\steamapps\common\Dyson Sphere Program\DSPGAME_Data\Managed\Assembly-CSharp.dll</HintPath>
```

Comment:
do not change this one

Reply:
Yeah I can't seem to ignore that file. It's in my gitignore, but keeps uploading

Reply:
I can show you how to circumcent this so you dont have to manually change the path everytime. I think I even talked to glitch about it on discord.